### DISCUSSION TITLE
Welcome to traitor Discussions!

### DISCUSSION THREAD

**Discussion by liamg** [liamg/traitor#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Well A very big no sleep congrats to you guys salaam my biggest respects dependabot head writers owners game overs well lost that address I will mail this Mac with that bot without postal charge

**Comment by TobiTenno** [comment]
Hey, I'm looking at implementing this into [fsociety-team/fsociety](https://github.com/fsociety-team/fsociety), would you be cool with that?

**Reply by liamg** [reply]
Of course, it would be an honour :)

**Reply by TobiTenno** [reply]
Cool! Its now added.
